core/src/CopyChip/CopyChip.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from "react";
+import cx from "classnames";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faClipboard, faCheck } from "@fortawesome/free-solid-svg-icons";
+
+import { Button } from "../Button";
+
+import styles from "./CopyChip.less";
+
+export const CopyChip = (props) => {
+  const [copied, setCopied] = useState(false);
+
+  function fallback(text) {
+    if (window.clipboardData && window.clipboardData.setData) {
+      // IE specific code path to prevent textarea being shown while dialog is visible.
+      return window.clipboardData.setData("Text", text);
+    } else if (
+      document.queryCommandSupported &&
+      document.queryCommandSupported("copy")
+    ) {
+      const textarea = document.createElement("textarea");
+      textarea.innerText = text;
+      document.body.appendChild(textarea);
```

Comment:
Why are we appending this to the document? Can we not select it's content without it being attached?

Reply:
In order for the method `.select()` to work properly the input/textarea needs to be appended into the dom. This is from reading online and referencing your example FieldTypeUUID